src/main/ngapp/accio/src/app/editor/mask.directive.ts

```diff
@@ -117,6 +129,14 @@ export class MaskDirective {
       // Fire event to draw pixel
       this.continuePaintEvent.emit(pixel);
     }
+
+    else if (this.tool == MaskTool.PAN && this.mouseDown) {
```

Comment:
nit: "else if" should be on the same line as the previous closing brace, e.g.:

   `} else if (this.tool == MaskTool.PAN && this.mouseDown) {`